packages/components/pages/f-mfa/src/components/Mfa.vue

```diff
@@ -189,14 +189,18 @@ export default {
 
         errorPrimaryButton () {
             return {
-                href: this.cleanUrl(this.returnUrl),
+                href: this.cleanUrl(this.returnUrl || '/'),
                 text: this.$t('errorMessages.loading.primaryButtonText')
             };
         }
     },
 
+    created () {
+        this.initialise(); // Query string validation should happen during SSR to prevent reflecting malicious input
+    },
+
     mounted () {
-        this.initialise();
+        this.otp = ''; // Clear the field
     },
```

Comment:
I've removed this need to clear the field now ... unsure why it's still appearing on your pr tho

Reply:
will fix on next pr, nw

Reply:
Probably snuck through a merge conflict